### ISSUE TITLE
`bcf` removed from CRAN

### ISSUE THREAD

**Issue by ngreifer** [jaredsmurray/bcf#20]
It looks `bcf` was removed from CRAN in April. Let me know if you would like some help getting it back. Based on the CRAN check archive, it looks like it would not be too hard to fix.

**Comment by jaredsmurray** [comment]
Yes, small fix but I haven't had the time to make it. I'd welcome a PR!
________________________________
From: Noah Greifer ***@***.***>
Sent: Wednesday, June 28, 2023 10:29 AM
To: jaredsmurray/bcf ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [jaredsmurray/bcf] `bcf` removed from CRAN (Issue #20)


It looks bcf was removed from CRAN in April. Let me know if you would like some help getting it back. Based on the CRAN check archive, it looks like it would not be too hard to fix.
